pkg/apis/autoheal/v1alpha2/zz_generated.conversion.go

```diff
@@ -0,0 +1,121 @@
+// +build !ignore_autogenerated
+
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by conversion-gen. DO NOT EDIT.
+
+// This file was autogenerated by conversion-gen. Do not edit it manually!
+
+package v1alpha2
+
+import (
+	unsafe "unsafe"
+
+	autoheal "github.com/openshift/autoheal/pkg/apis/autoheal"
```

Comment:
why do we need to use explicit package names here ?
can we just do ?
```
import (
    "unsafe"
    "github.com/openshift/autoheal/pkg/apis/autoheal"        
    ...
```

what am i missing here ?